compose/ui/ui-text/src/nativeMain/kotlin/androidx/compose/ui/text/intl/NativePlatformLocale.native.kt

```diff
@@ -20,19 +20,21 @@ import platform.Foundation.countryCode
 import platform.Foundation.currentLocale
 import platform.Foundation.languageCode
 import platform.Foundation.NSLocale
+import platform.Foundation.localeIdentifier
 import platform.Foundation.scriptCode
 
 internal class NativeLocale(val locale: NSLocale) : PlatformLocale {
     override val language: String
-        get() = locale.languageCode!!
+        get() = locale.languageCode
```

Comment:
does it need to be `locale.languageCode.orEmpty()` like with `scriptCode` below?

Reply:
LanguageCode returns a strict `String` type.
While scriptCode and countryCode return nullable `String?`

So, I decided to remove this redundant `!!`